Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -201,15 +202,18 @@ class AttachmentSaveEventListener final : public EventListener {
     void handleEvent(ScriptExecutionContext&, Event& event) final
     {
         if (event.type() == eventNames().clickEvent) {
+            ASSERT(event.isMouseEvent());
+            auto& mouse = static_cast<MouseEvent&>(event);
```

Comment:
```suggestion
            auto& mouseEvent = downcast<MouseEvent>(event);
```
downcast also covers the assert.